debian_slim/Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM debian:jessie
+ARG CLOUD_SDK_VERSION=160.0.0
```

Comment:
can we avoid having a value here. Even though it's an ARG, I don't think something hardcoded is good, we would need to keep it up to date.